image/templates/common/port-forward.sh

```diff
@@ -16,20 +16,22 @@
 #     $ ./port-forward.sh 8443
 
 if [ -z "$1" ]; then
-	echo "usage: $0 8443"
+	echo "usage: $0 8443 [namespace]"
```

Comment:
```suggestion
	echo "usage: $0 [port] [namespace]"
```

(as far as I understand, port number is configurable as first input parameter to the script)

Reply:
done